client-grpc/examples/grpc.rs

```diff
@@ -88,6 +90,152 @@ async fn get_itineraries() -> Result<itinerary::List, Status> {
     }
 }
 
+/// Example AdsbClient
+/// Assuming the server is running, this method calls `client.itineraries` and
```

Comment:
Incorrect documentation. This method calls the AdsbClient, not itineraries

Reply:
Fixed with https://github.com/Arrow-air/svc-storage/pull/45/commits/b5715a1403023460814b2f94c7a84306ab603df2